src/tests/0000ztrips.js

```diff
@@ -390,4 +399,13 @@ describe('trips tests', () => {
         done();
       });
   });
+  it('should return the number of trips created by a user', (done) => {
+    const startDate = '2020-01-30';
+    chai.request(app).get(`/api/v1/trip-statistics?startDate=${startDate}`)
+      .set('token', `Bearer ${token2}`)
+      .end((err, res) => {
+        res.should.have.status(200);
```

Comment:
@jo-rdan You need not only to assert the response status but also the returned data. Test if the correct data are returned.